### PR TITLE
Origin/docs improvements

### PR DESCRIPTION
**Pull request by cjh84**

### CODE REVIEW

File: _docs/install/clusterinstall.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ $ git clone https://github.com/andrelucas/storageos-alpine.git
 $ cd storageos-alpine
 ```
 
-2. Bring up a three-node StorageOS cluster.
+2. Bring up a three-node StorageOS cluster. This takes around five minutes.
```
Comment (rusenask) [inline _docs/install/clusterinstall.md]: probably depends on machine size

File: _docs/manage/labels.md [hunk 2]
```diff
@@ -3,56 +3,102 @@ layout: guide
...
 # Labels
 
-Labels are a mechanism for applying metadata to StorageOS objects. You can use them to annotate or organise your volumes in any way that makes sense for your organization or app.
+Labels are a mechanism for applying metadata to StorageOS objects. You can use
+them to annotate or organise your volumes in any way that makes sense for your
+organization or app.
 
-A label is a key-value pair that is stored as a string. An object may have multiple labels but each key-value pair must be unique within an object.
+A label is a key-value pair that is stored as a string. An object may have
+multiple labels but each key-value pair must be unique within an object.
```
Comment (rusenask) [inline _docs/manage/labels.md]: actually only the keys have to be unique since it's a simple JSON

File: _docs/manage/rules.md [hunk 3]
```diff
@@ -11,11 +11,11 @@ Rules apply custom configuration to volumes based on matched labels.
 
 ## Overview
 
-Rules are used for managing data policy and placement using StorageOS *features* such as replication, QoS and compression.
+Rules are used for managing data policy and placement using StorageOS features such as replication, QoS and compression.
 
-Rules are created using *labels* and *operators* to evaluate what StorageOS *feature* to apply.  Where multiple rules apply to the same label, a *weight* is used to determine the order of evaluation.
+Rules are created using [selectors](selectors.html) and [labels](labels.html) to evaluate what StorageOS feature to apply.
```
Comment (rusenask) [inline _docs/manage/rules.md]: didn't know that you can make links like this, cool!

File: _docs/manage/labels.md (hunk 2 above)
Comment (cjh84) [inline _docs/manage/labels.md]: Fixed.

File: _docs/install/clusterinstall.md (hunk 1 above)
Comment (cjh84) [inline _docs/install/clusterinstall.md]: It's just an estimate, so I'll leave it.

**Pull request merged**